test/e2e/run_networking_test.go

```diff
@@ -178,4 +178,31 @@ var _ = Describe("Podman run networking", func() {
 		Expect(exec4.ExitCode()).To(Equal(0))
 		Expect(exec4.OutputToString()).To(ContainSubstring("192.0.2.2 test1"))
 	})
+
+	It("podman run network in user created network namespace", func() {
+		SystemExec("ip", []string{"netns", "add", "xxx"})
+		session := podmanTest.Podman([]string{"run", "-dt", "--net", "ns:/run/netns/xxx", ALPINE, "wget", "www.projectatomic.io"})
```

Comment:
should we change projectatomic.io to podman.io?

Reply:
Fixed